modules/localgov_workflows/localgov_workflows.info.yml

```diff
@@ -0,0 +1,9 @@
+name: LocalGovDrupal Workflows
+description: Provides the default editorial workflow for content types.
+package: LocalGovDrupal
+type: module
+core_version_requirement: ^8.8 || ^9
+
+dependencies:
+  - drupal:content_moderation
+  - localgovdrupal:localgov_core
```

Comment:
Just a question more than anything else, what should the namespace of a dependency be? This works fine, but I would have used `main_module:sub_module` so `localgov_core:localgov_core`. Whatever we decide we should be consistent in it's usage.

Reply:
Think I was just getting confused between composer dependencies and drupal dependencies. It is `main_module:sub_module` so `localgov_core:localgov_core`